src/containers/augmintToken/index.js

```diff
@@ -34,60 +34,71 @@ class AugmintToken extends React.Component {
 
     render() {
         return (
-            <EthereumState>
-                <Psegment>
-                    <TopNavTitlePortal>
-                        <FeatureContext>
-                            {features => features.dashboard ? <Pheader className="secondaryColor" header="Reserves" /> : null}
-                        </FeatureContext>
-                    </TopNavTitlePortal>
-                    <Pgrid.Row>
-                        <Pgrid.Column>
-                            <TotalSupply
-                                augmintToken={this.props.augmintToken}
-                                monetarySupervisor={this.props.monetarySupervisor}
-                            />
-                        </Pgrid.Column>
-                        <TopNavTitlePortal>
-                            <FeatureContext>
-                                {features => features.dashboard ? null : <Pheader header="Reserves" style={{ width: "100%" }} />}
-                            </FeatureContext>
-                        </TopNavTitlePortal>
-                        <Pgrid.Column>
-                            <ReserveStats
-                                augmintToken={this.props.augmintToken}
-                                monetarySupervisor={this.props.monetarySupervisor}
-                                rates={this.props.rates}
-                            />
-                        </Pgrid.Column>
+            <FeatureContext>
+                {features => {
+                    const dashboard = features.dashboard;
+                    return (
+                        <EthereumState>
+                            <Psegment>
+                                <TopNavTitlePortal>
+                                    {dashboard && <Pheader className="secondaryColor" header="Reserves" />}
+                                </TopNavTitlePortal>
+                                <Pgrid.Row>
+                                    <Pgrid.Column>
+                                        <TotalSupply
+                                            augmintToken={this.props.augmintToken}
+                                            monetarySupervisor={this.props.monetarySupervisor}
+                                        />
+                                    </Pgrid.Column>
+                                    <TopNavTitlePortal>
```

Comment:
you can remove the portal wrapper from here, this header doesn't get ported to the topNav

Reply:
👍